common.blocks/menu-item/menu-item.bemhtml

```diff
@@ -2,7 +2,20 @@ block('menu-item')(
     js()(function() {
         return { val : this.ctx.val };
     }),
-    attrs()({ role : 'menuitem' }),
+    attrs()(function(){
+        var mods = this.mods,
+            menuMode = this._menuMods && this._menuMods.mode,
+            role = menuMode?
+                        (menuMode === 'check'? 'menuitemcheckbox' : 'menuitemradio') :
+                        'menuitem',
+            attrs = {
+                role : role,
+                'aria-disabled' : mods.disabled,
```

Comment:
If you talking about `_disabled_false` case — we don't need to support it. See [documentation](https://en.bem.info/libs/bem-components/v2.2.1/desktop/menu-item/#disabled) for details